lib/cleanup.py

```diff
@@ -13,9 +13,9 @@ def floatable(value):
 def correct_invalid_value(value, args):
     """This cleanup function replaces null indicators with None."""
```

Comment:
lets add missing values to 
"""This cleanup function replaces null indicators with None.""">> """This cleanup function replaces missing value indicators with None."""